QuickSort.js

```diff
@@ -0,0 +1,23 @@
+function quickSort(arr) {
```

Comment:
https://www.w3resource.com/javascript-exercises/searching-and-sorting-algorithm/searching-and-sorting-algorithm-exercise-1.php

Reply:
bad copying, function doesn't work